source/docs/08-lyricsperfdir.xml

```diff
@@ -200,7 +200,7 @@
                   <figure>
                      <head/>
                      <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve" valid="feasible">
-                        <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/lyricsDesc/vagabond.txt" parse="text"/>
+                        <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/lyricsDesc/lyricsDesc-sample284.txt" parse="text"/>
```

Comment:
Why rename this example? I think it's better to have speaking names instead of these generic titles.

Reply:
Since I work on the guidelines I started to follow the "naming system" I find. All the other samples here are named like this. Is there a coding guideline for file naming I missed?